Xray Plug does not work in IDEA 2020.1

**Describe the bug**
Xray Plugin does not work in IDEA 2020.1, with Error:
Exception: This plugin is not a production of JetBrains.

com.intellij.diagnostic.PluginException: org/jetbrains/plugins/groovy/lang/psi/GroovyElementVisitor [Plugin: org.jfrog.idea]
	at com.intellij.serviceContainer.ComponentManagerImpl.instantiateExtensionWithPicoContainerOnlyIfNeeded(ComponentManagerImpl.kt:663)
	at com.intellij.serviceContainer.BaseKeyedLazyInstance.createInstance(BaseKeyedLazyInstance.java:57)
	at com.intellij.openapi.extensions.CustomLoadingExtensionPointBean.createInstance(CustomLoadingExtensionPointBean.java:31)
	at com.intellij.codeInspection.InspectionEP.instantiateTool(InspectionEP.java:198)
	at com.intellij.codeInspection.ex.InspectionToolWrapper.getTool(InspectionToolWrapper.java:72)
	at com.intellij.codeInsight.daemon.impl.LocalInspectionsPass.getInspectionTools(LocalInspectionsPass.java:728)
	at com.intellij.codeInsight.daemon.impl.LocalInspectionsPass.collectInformationWithProgress(LocalInspectionsPass.java:115)
	at com.intellij.codeInsight.daemon.impl.ProgressableTextEditorHighlightingPass.doCollectInformation(ProgressableTextEditorHighlightingPass.java:84)
	at com.intellij.codeHighlighting.TextEditorHighlightingPass.collectInformation(TextEditorHighlightingPass.java:52)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.lambda$null$1(PassExecutorService.java:434)
	at com.intellij.openapi.application.impl.ApplicationImpl.tryRunReadAction(ApplicationImpl.java:1106)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.lambda$doRun$2(PassExecutorService.java:427)
	at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:625)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:570)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:61)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.doRun(PassExecutorService.java:426)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.lambda$run$0(PassExecutorService.java:402)
	at com.intellij.openapi.application.impl.ReadMostlyRWLock.executeByImpatientReader(ReadMostlyRWLock.java:168)
	at com.intellij.openapi.application.impl.ApplicationImpl.executeByImpatientReader(ApplicationImpl.java:168)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.run(PassExecutorService.java:400)
	at com.intellij.concurrency.JobLauncherImpl$VoidForkJoinTask$1.exec(JobLauncherImpl.java:171)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:290)
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1020)
	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1656)
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1594)
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:177)
Caused by: java.lang.NoClassDefFoundError: org/jetbrains/plugins/groovy/lang/psi/GroovyElementVisitor
	at java.base/java.lang.Class.forName0(Native Method)
	at java.base/java.lang.Class.forName(Class.java:398)
	at com.intellij.serviceContainer.ComponentManagerImpl.instantiateExtensionWithPicoContainerOnlyIfNeeded(ComponentManagerImpl.kt:660)
	... 25 more
Caused by: java.lang.ClassNotFoundException: org.jetbrains.plugins.groovy.lang.psi.GroovyElementVisitor PluginClassLoader[org.jfrog.idea, 1.6.0] com.intellij.ide.plugins.cl.PluginClassLoader@1488f3e3
	at com.intellij.ide.plugins.cl.PluginClassLoader.loadClass(PluginClassLoader.java:115)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:521)
	... 28 more

**To Reproduce**
1. Start IDEA
2. Install Xray plugin
3. Connect to Xray Server
4. Error: Plugin does not work,  and can find error message in IDE Fatal Errors

**Expected behavior**
The pluging should scan project.


**Versions**
- JFrog IDEA plugin version:  1.6.0
- Operating system:  Mac OS 10.14.6
- Xray version:  3.2.0
- IDEA: IntelliJ IDEA 2020.1 (Ultimate Edition)

--- a/src/main/resources/META-INF/plugin.xml
+++ b/src/main/resources/META-INF/plugin.xml
@@ -21,6 +21,8 @@
     <idea-version since-build="183.3795.13"/>
     <depends>com.intellij.modules.lang</depends>
     <depends config-file="with-java.xml" optional="true">com.intellij.modules.java</depends>
+    <depends config-file="with-gradle.xml" optional="true">org.jetbrains.plugins.gradle</depends>
+    <depends config-file="with-maven.xml" optional="true">org.jetbrains.idea.maven</depends>
     <depends config-file="with-go.xml" optional="true">org.jetbrains.plugins.go</depends>
 
     <application-components>

--- a/src/main/resources/META-INF/with-java.xml
+++ b/src/main/resources/META-INF/with-java.xml
@@ -1,4 +1,1 @@
-<idea-plugin>
-    <depends config-file="with-gradle.xml" optional="true">org.jetbrains.plugins.gradle</depends>
-    <depends config-file="with-maven.xml" optional="true">org.jetbrains.idea.maven</depends>
-</idea-plugin>
+<idea-plugin/>

